test/test_workspaces.py

```diff
@@ -41,3 +42,18 @@ def test_order_paths(self):
         self.assertEqual(['foo', 'bar'], order_paths(['bar', 'foo'], ['foo']))
         self.assertEqual(['baz', 'foo', 'bar'], order_paths(['bar', 'foo', 'baz'], ['baz', 'foo']))
         self.assertEqual(['foo' + os.sep + 'bim', 'bar'], order_paths(['bar', 'foo' + os.sep + 'bim'], ['foo']))
+
+    @unittest.skipUnless(sys.platform.startswith('linux'), 'requires Unix')
```

Comment:
Instead of checking the platform (e.g. it should also work on macOS) maybe check if the symlink can be created?

Reply:
That's a good idea. Afaik `os.symlink` is simply not present when unavailable, let's use `hasattr` in skip condition.

Reply:
Instead of checking for existence please check for an exception when invoking the function instead.

Reply:
By calling function in skip condition expression I'd be creating a symlink, so I assume you recommend to forgo `skipUnless` in favor of silently passing the test by catching `AttributeError` when `symlink` attribute is not found in `os` module in test body?

Reply:
I settled for:
```
            try:
                os.symlink(foo, foo_ln)
            except (AttributeError, OSError):
                self.skipTest('requires symlink availability')
```